apps/emqx_bridge_pulsar/test/emqx_bridge_pulsar_v2_SUITE.erl

```diff
@@ -275,7 +275,7 @@ start_consumer(TestCase, Config) ->
         cacertfile => filename:join([CertsPath, "cacert.pem"])
     },
     Opts = #{enable_ssl => UseTLS, ssl_opts => emqx_tls_lib:to_client_opts(SSLOpts)},
-    {ok, _ClientPid} = pulsar:ensure_supervised_client(ConsumerClientId, [URL], Opts),
+    {ok, _} = pulsar:ensure_supervised_client(ConsumerClientId, [URL], Opts),
```

Comment:
It is not pid anymore?

Reply:
It is, but I went back and forth in changing the driver while testing with EMQX, ended up changing the variable binding name.